py3status/modules/sysdata.py

```diff
@@ -195,16 +195,20 @@ def _get_stat(self):
                 'idle': int(fields[4]),
             }
 
-    def _calc_mem_info(self, unit='GiB', memi=dict, keys=list):
+    def _calc_mem_info(self, unit='GiB', memi=dict, keys=list, mem=bool):
```

Comment:
`memi=dict, keys=list` can we change these to  `memi=None, keys=None` because dict/list makes no sense

Reply:
Minimal diff is the game here. I'm removing them all.